ff2mpv.js

```diff
@@ -21,5 +32,6 @@ browser.contextMenus.onClicked.addListener((info, tab) => {
 });
 
 browser.browserAction.onClicked.addListener((tab) => {
+    notifyContentScript();
```

Comment:
Let's move this call inside of the body of `ff2mpv`; that way activation via the context menu button also correctly pauses videos.